src/shared/components/entry/SequenceView.tsx

```diff
@@ -94,7 +94,7 @@ export const SequenceInfo = ({
       content: lastUpdateDate,
     },
     {
-      title: 'Checksum',
+      title: <span data-article-id="checksum">Checksum</span>,
```

Comment:
Would be good to update the corresponding types in franklin. See TypeScript error.

Reply:
Updated here https://github.com/ebi-uniprot/franklin-sites/pull/238/commits/312b44112db5ffd1cddddb418d380e97a3010829